### ISSUE TITLE
Documentation mentioning `{runtime}` is confusing

### ISSUE THREAD

**Issue by altano** [remix-run/remix#3007]
### What version of Remix are you using?

1.4.1

### Steps to Reproduce

A lot of the recommendation uses `@remix-run/{runtime}` which I think is pretty confusing, e.g. https://remix.run/docs/en/v1/api/remix#json has this code:

```js
import { json } from "@remix-run/{runtime}";
```

It looks like this should be `import { json } from "@remix-run/node";` if your runtime is node but this was not at all immediately clear to me.

### Expected Behavior

Perhaps the documentation could either default to `@remix-run/node` and mention other runtimes, or every line that has such an import could also include a trailing comment explaining it?

### Actual Behavior

Documentation trips up beginners like me?

**Comment by achinchen** [comment]
I've the same feelings at first glance!

**Comment by kiliman** [comment]
Yes, I'm sure the team will work on cleaning up the documentation. This was a big change in philosophy (although not a breaking change, since the old way still works).

Docs should be something like:
```ts
import { json } from '@remix-run/node'  // or '@remix-run/cloudflare' etc
```

Perhaps a future version of the docs will give you a selector: 

- Language: TypeScript
- Framework: React
- Platform: Node

Then it'll automatically present the docs accordingly.
